webpack.config.js

```diff
@@ -1,20 +1,20 @@
-import fastGlob from 'fast-glob';
+import webpack from 'webpack';
 import wrapAnsi from 'wrap-ansi';
+import fastGlob from 'fast-glob';
+import VueLoader from 'vue-loader';
+import { fileURLToPath } from 'url';
```

Comment:
Spaces after and before curl brackets would require linter configuration changes

Reply:
I think we do not need to keep the spaces. Maybe it's better to stick to `import {fileURLToPath} from 'url';`

Reply:
Sure, no worries. It's just a good formatting practice in JS/TS to keep these things separated with a space. But yeah I wouldn't bother either, since it's only just a configuration file